src/averageAges.js

```diff
@@ -37,7 +52,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const mothers = people.map(person => person.mother);
+  let women = people.filter(person => person.sex === 'f');
```

Comment:
here the same: you can avoid one more filter, if you will write condition with ternary operator here, in 
 const women = ...
 if the person is someone's mother - her sex exactly === 'f'